libopenage/assetmanager.cpp

```diff
@@ -17,10 +17,11 @@
 
 namespace openage {
 
-AssetManager::AssetManager(util::Dir *root)
+AssetManager::AssetManager(gui::GuiItemLink *gui_link)
 	:
-	root{root},
-	missing_tex{nullptr} {
+	root{std::string()},
```

Comment:
Assumes current directory. Then gui figures out what to set. For now, it's taken from the command line arguments.